README.md

```diff
@@ -11,65 +11,85 @@
 
 ## Usage
-As mentioned above this repository is a plugin for VulcanizeDB.  As such it cannot be run as a standalone executable,
-but instead is intended to be included as part of a VulcanizeDB core command. There are two VulcanizeDB core commands that 
-are required for events and storage slots to be transformed and persisted to the Postgres database.
+As mentioned above this repository is a plugin for VulcanizeDB.  As such it cannot be run as a standalone executable, but instead is intended to be included as part of a VulcanizeDB core command. There are two VulcanizeDB core commands that are required for events and storage slots to be transformed and persisted to the Postgres database.
 
-1. `headerSync` fetches raw Ethereum data and syncs it into VulcanizeDB's Postgres database where then it can be used for
-transformations. More information about the `headerSync` command can be found in the [VulcanizeDB repository](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/data-syncing.md#headersync).
+1. `headerSync` fetches raw Ethereum data and syncs it into VulcanizeDB's Postgres database where then it can be used for transformations. More information about the `headerSync` command can be found in the [VulcanizeDB repository](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/data-syncing.md#headersync).
 
-1. `execute` uses the raw Ethereum data that has been synced into Postgres and applies transformations to configured MCD
-contract data via [event](./transformers/events) and [storage](./transformers/storage) transformers. The VulcanizeDB
-repository includes a [general description](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/custom-transformers.md)
-about transformers.
+1. `execute` uses the raw Ethereum data that has been synced into Postgres and applies transformations to configured MCD contract data via [event](./transformers/events) and [storage](./transformers/storage) transformers. The VulcanizeDB repository includes a [general description](https://github.com/makerdao/vulcanizedb/blob/staging/documentation/custom-transformers.md) about transformers.
 
-These core commands can be run via Docker images which is the preferred method, or they can be built and run via the
-command line interface. In either method, a postgres database will first need to be created:
+The core commands can also be run via images or built and run via the command line interface. In either method, a postgres database will first need to be created:
 1. Install Postgres
-1. Create a user for yourself that is able run migrations and add extensions.
+   1. Create a user for yourself that is able run migrations and add extensions.
 1. `createdb vulcanize_public`
-1. Migrating the database manually is unnecessary as the commands handle this.
+1. Migrate the database using the `make migrate` task in this repository.
```

Comment:
fwiw I believe `execute` and `composeAndExecute` _do_ handle running the migrations for you, but if you went that path you would need to be sure to run those commands _first_ since the vulcanizedb migrations alone will moot out migrations in this repo.

Reply:
You're correct - I changed the directions for this reason. It might be good to re-warn the reader here of what will happen if you do things in the wrong order.